components/Tabs/Tabs.js

```diff
@@ -3,49 +3,57 @@ class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
     // this.element;
-    
+    this.element = element;
     // Get the custom data attribute on the Link
     // this.data;
-    
+    this.data = this.element.dataset.tab;
     // Using the custom data attribute get the associated Item element
     // this.itemElement;
-    
+    this.itemElement = document.querySelector(`.tabs-item[data-tab="${this.data}"]`);
     // Using the Item element, create a new instance of the TabItem class
     // this.tabItem;
-    
+    this.tabItem = new TabItem(this.itemElement);
     // Add a click event listener on this instance, calling the select method on click
-
+    this.element.addEventListener('click', () => {
+      this.select();
+    })
+    this.element.addEventListener('click', () => this.fade());
   };
 
   select() {
     // Get all of the elements with the tabs-link class
     // const links;
-
+      const links = document.querySelectorAll('.tabs-link');
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
     // Array.from(links).forEach();
-
+      links.forEach( link => link.classList.remove('tabs-link-selected'));
     // Add a class named "tabs-link-selected" to this link
     // this.element;
-    
+      this.element.classList.add('tabs-link-selected');
     // Call the select method on the item associated with this link
-
+    this.tabItem.select();
+  }
+  fade() {
+    $(function () {
+      $('.tabs-item-selected').hide().fadeIn(2000);
```

Comment:
Would this be some jquery? Very impressive. I don't know anything about jquery, but I know it is still pretty prevalent in a lot of websites. I believe it is starting to fall out of favor now that there are like a million animation libraries for every javaScript framework. The more you know about legacy code, the more valuable you will be to a company looking to modernize their stack.